index.js

```diff
@@ -159,7 +165,13 @@ const init = () => {
 
 init()
 
+process.on('unhandledRejection', (reason, p) => {
+  // Otherwise unhandled promises are not possible to trace with the information logged
+  console.log('Unhandled Rejection at: Promise', p, 'reason:', reason, 'error stack:', reason.stack);
```

Comment:
Can we make the whole process exit in this case? That will make k8s restart the script. If we don't do this, the process could become stuck and don't do anything.